examples/fips/README.md

```diff
@@ -0,0 +1,83 @@
+# FIPS Compliance
+
+We are testing and making this client FIPS compliant using OpenSSL 3.0. Use OpenSSL 3.0 to use this client in FIPS compliant mode. Older versions of OpenSSL might work in similar way but we have not verified that.
+
+## Communication between client and Kafka cluster
+
+### Install from prebuilt wheels
+
+If this client is installed using `pip`, it comes with librdkafka in which OpenSSL 3.0 is statically linked. This client can communicates with Kafka cluster with FIPS approved algorithms using FIPS provider provided by OpenSSL. Steps to enable FIPS provider is mentioned in section [Enabling FIPS provider](#enabling-fips-provider).
+
+
+### Install from source by building librdkafka
+
+When building librdkafka and python client from source, librdkafka uses OpenSSL present in the system by dynamically linking it. If the installed OpenSSL is already working in FIPS mode, then you can directly jump to section [Client configuration to enable FIPS provider](#client-configuration-to-enable-fips-provider), otherwise use steps mentioned in section [Enabling FIPS provider](#enabling-fips-provider) to make system OpenSSL FIPS compliant. Once that is done, librdkafka and python client will use FIPS approved algorithms for the communication between client and Kafka Cluster using producer, consumer or admin client.
```

Comment:
```suggestion
When you build the python client from source, librdkafka dynamically links to the OpenSSL present in the system. If the installed OpenSSL is already working in FIPS mode, then you can directly jump to section [Client configuration to enable FIPS provider](#client-configuration-to-enable-fips-provider), otherwise use steps mentioned in section [Enabling FIPS provider](#enabling-fips-provider) to make OpenSSL in your system FIPS compliant. Once that is done, librdkafka and python client will use FIPS approved algorithms for the communication between client and Kafka Cluster using producer, consumer or admin client.
```

Reply:
Used your suggestion with some changes. Please check.

Reply:
The change to "When you build the python client from source" is ok because one could choose to install the binary librdkafka linux package and just compile or install OpenSSL. Probably it's the better alternative to just using the wheels.